sos/report/plugins/__init__.py

```diff
@@ -134,32 +134,30 @@ def __str(self, quote=False, prefix="", suffix=""):
             optional prefix, suffix and value quoting.
         """
         quotes = '"%s"'
```

Comment:
Technically we will still use "s-string" due to this (but I think it does not matter?).

Reply:
Unless we make other wholesale changes.

At the moment I am looking to convert based on `pylint`, maybe we can have a another one later once parts 1, 2 and 3(in-coming) get merged